### PR TITLE
Child validation update

### PR DESCRIPTION
**Pull request by HernandezM-dev**
# Description

Fixes #

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by HernandezM-dev** [comment]
Minor fixes to rest operations and their validations

**Review by saharj — APPROVED** [review]
Awesome!

File: api/cronTasks/cronTasks.js [hunk 1]
```diff
@@ -17,6 +17,18 @@ const addTotalPointsToChildren = async () => {
     }
 }
 
+const resetTable = async(table) =>{
+    const IDs = await getIDs(table)
+    await deleteAll(table, IDs)
+}
+
+const deleteAll = async(table, IDs) =>{
+    console.log(IDs)
```
Comment (Jahteo) [inline api/cronTasks/cronTasks.js:26]: I like leaving console logs in code, but make sure that they're commented out for PR's & ideally labeled too. Something like: `console.log("cronTasks ID's from deleteAll: -------->", IDs)` or however you wanna label & format is always appreciated.

**Review by Jahteo — APPROVED** [review]
Love how you're adding such key functionality for current and future devs in a clean and straightforward code!

**Pull request merged**